Frontend/src/pages/registration_edit/components/RegistrationEditor.jsx

```diff
@@ -151,7 +158,31 @@ export default function RegistrationEditor() {
               checked={status === 'cancelled'}
               onChange={(_, data) => setStatus(data.value)}
             />
+            <br />
+            <Header>Guests</Header>
+            <Input
+              disabled={registrationEditDeadlinePassed}
+              type="number"
+              min={0}
+              max={99}
```

Comment:
Doesn't the competition information include the maximum number of guests allowed (and whether guests are allowed at all)?